projects/cscope.sourceforge.io/package.yml

```diff
@@ -0,0 +1,30 @@
+distributable:
+  url: https://downloads.sourceforge.net/project/cscope/cscope/v{{version.marketing}}/cscope-{{version.marketing}}.tar.gz
+  strip-components: 1
+
+versions:
+  - 15.9
```

Comment:
Looks like this could be:
```yaml
versions:
  url: https://sourceforge.net/projects/cscope/files/cscope/
  match: _/projects/cscope/files/cscope/\d+.\d+/_
  strip:
    - _^/projects/cscope/files/cscope/_
    - _/$_
```

Reply:
I already tried

mirror: 404: https://dist.tea.xyz/cscope.sourceforge.io/v15.8.0.tar.gz
404: https://downloads.sourceforge.net/project/cscope/cscope/v15.8/cscope-15.8.tar.gz
tea expands the full semantic version, which may mean the URL you are
fetching is now incorrect. Try `version.raw`?
andrejrabcenko@MacBook-Pro-Andrej pantry %

Reply:
for example:
v 15.8 tar.bz2
v 15.9 tar.gz

Reply:
looks like they bounced back and forth between gz and bz2 for different releases. no way to handle _that_ yet. A complete solution probably uses:
- `{{ version.raw }}` rather than `{{ version.marketing }}` to hand e.g. `15.8b`
- ignore: [15.8, 15.7a, 15.7] # provided .bz2 rather than .gz